docs/enhancements/arm64-cpu-architecture.md

```diff
@@ -0,0 +1,149 @@
+---
+title: arm64-cpu-architecture
+authors:
+  - "@danielerez"
+creation-date: 2021-07-29
+last-updated: 2021-07-29
+---
+
+# ARM64 CPU architecture support
+
+## Summary
+
+The default host CPU architecture supported for installation is x86_64 in Assisted Installer. In order to support additional CPU architectures, the assisted-service should accept and handle selection of arm64 CPU architecture as well.
+
+## Motivation
+
+Although x86_64 is popular in many use cases, other CPU architectures are common in other segments.  For example, ARM is popular in edge use cases.  OpenShift has recently added support for arm64 and it is important to add this support to Assisted Installer.
+
+### Goals
+
+* Allow installation of clusters that are running on arm64 hosts.
+* The introduced infrastructure should facilitate future support of more architectures (e.g. PPC).
+
+### Non-Goals
+
+* The assisted-service will remain for x86_64 only.
+* No support for clusters with hosts of multiple architectures (e.g. x86_64 + arm64).
+* Implementation for kube-api (though design should take into account).
```

Comment:
Unsure of what you mean by this?

Reply:
Just that changes in ClusterImageSet CRD and the operator are out of the scope in this phase.